### PR TITLE
Rename Makefile targets following cookiecutter-volto#14

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: frontend_addon/{{ cookiecutter.__folder_name }}/Makefile [hunk 1]
```diff
@@ -47,57 +55,65 @@ format: ## Format codebase
...
-.PHONY: start-test-acceptance-server
-start-test-acceptance-server: ## Start acceptance server
+.PHONY: acceptance-backend-start
+acceptance-backend-start: ## Start backend acceptance server
 	docker run -it --rm -p 55001:55001 $(DOCKER_IMAGE_ACCEPTANCE)
 
-.PHONY: start-test-acceptance-server-ci
-start-test-acceptance-server-ci: ## Start acceptance server in CI mode (no terminal attached)
+.PHONY: ci-acceptance-backend-start
+ci-acceptance-backend-start: ## Start acceptance server in CI (no terminal attached)
```
Comment (stevepiercy) [inline frontend_addon/{{ cookiecutter.__folder_name }}/Makefile:110]:
```suggestion
ci-acceptance-backend-start: ## Start backend acceptance server in headless mode for CI
```

I also added this to https://github.com/plone/cookiecutter-volto/issues/14#issuecomment-2068663318

File: frontend_addon/{{ cookiecutter.__folder_name }}/Makefile [hunk 2]
```diff
@@ -47,57 +55,65 @@ format: ## Format codebase
...
 
-.PHONY: test-acceptance
-test-acceptance: ## Start Cypress in interactive mode
+.PHONY: acceptance-test
+acceptance-test: ## Start Cypress in interactive mode
 	pnpm --filter @plone/volto exec cypress open --config-file $(CURRENT_DIR)/cypress.config.js --config specPattern=$(CURRENT_DIR)'/cypress/tests/**/*.{js,jsx,ts,tsx}'
 
-.PHONY: test-acceptance-headless
-test-acceptance-headless: ## Run cypress tests in headless mode for CI
+.PHONY: acceptance-headless-test
```
Comment (stevepiercy) [inline frontend_addon/{{ cookiecutter.__folder_name }}/Makefile:117]: For discussion: should we rename this to be consistent with the other two `ci-*` targets?

```suggestion
.PHONY: ci-acceptance-test
```

File: frontend_addon/{{ cookiecutter.__folder_name }}/Makefile [hunk 3]
```diff
@@ -47,57 +55,65 @@ format: ## Format codebase
...
+acceptance-headless-test: ## Run cypress tests in headless mode for CI
```
Comment (stevepiercy) [inline frontend_addon/{{ cookiecutter.__folder_name }}/Makefile:118]: See previous comment.

```suggestion
ci-acceptance-test: ## Run cypress tests in headless mode for CI
```

File: frontend_addon/{{ cookiecutter.__folder_name }}/Makefile (hunk 1 above)
Comment (stevepiercy) [reply frontend_addon/{{ cookiecutter.__folder_name }}/Makefile:110]: Note I used "headless mode" in the target's description for consistency with other `ci-*` targets.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Looks great. However a new pattern emerged with `ci-*` targets, which needs agreement.

I also added the new `ci-acceptance-backend-start` target to https://github.com/plone/cookiecutter-volto/issues/14#issuecomment-2068663318. This addition established the `ci-*` pattern for targets.

**Comment by sneridagh** [comment]
@stevepiercy done!

**Pull request merged**